izer/backend/max7800x.py

```diff
@@ -3267,8 +3267,15 @@ def run_eltwise(
                 assets.from_template('assets', 'embedded-riscv-ai' + str(device), base_directory,
                                      test_name, board_name)
             else:
-                assets.from_template('assets', 'embedded-ai' + str(device), base_directory,
```

Comment:
Leave this call, and remove the templateMakefile. People might want to add other files, plus it's documented this way.